src/cloud_run/utils/constants.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+PROJECT_ID = "playground-413722"  # os.getenv("PROJECT_ID")
+SUBSCRIPTION_ID = os.getenv("SUBSCRIPTION_ID")
+DATASET_ID = "tollway_traffic"  # os.getenv("DATASET_ID")
```

Comment:
or `os.getenv("DATASET_ID", "tollway_traffic")`?

Reply:
this commit/change shouldn't have made it in. I have the values as literals right now to help with testing. Once I'm further along, if possible, thinking to define environment variables in Terraform script.